### ISSUE TITLE
Helpful message for create command when param is missing

### ISSUE THREAD

**Issue by vladimirvivien** [emccode/polly#82]
When creating a volume, if a param is missing, polly simply errors out.  Instead it should provide helpful message indicating the source of error.

The following shows what happen when I tried to create a volume with Polly cli tool.
```
polly volume create --name=testvol --servicename=swarm_virtualbox -l debug
DEBU[0000] updated log level                             logLevel=debug
FATA[0000] http error                                    status=500
```
The problem was that the size param is missing.

**Comment by clintonskitson** [comment]
@dvonthenen Would be great to loop this fix in for 0.1.1.

**Comment by dvonthenen** [comment]
This issue should be fixed now. If you are still seeing this, please reopen or file a new bug.
